src/platform/forms-system/src/js/web-component-patterns/fullNamePattern.js

```diff
@@ -79,6 +79,41 @@ const fullNameNoSuffixUI = (formatTitle, uiOptions = {}) => {
   };
 };
 
+/**
```

Comment:
can you duplicate all of the JSDocs for this function and make any changes? otherwise you don't get the information when you hover

Reply:
Will do.  I'll get that in for the Part 3 PR soon.